provisioning/transport/amqp/src/ParameterNames.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.Provisioning.Client.Transport.Amqp
+{
+    internal static class ParameterNames
+    {
+        public const string ForceRegistration = "forceRegistration";
```

Comment:
This is used in a single other place - any reason for this being declared in a different class?